_includes/configuration.md

```diff
@@ -68,7 +71,7 @@ necessary to alter the `max_connections` setting.
 
 **Note:**
 
-  - {{ site.data.product.title_short }} requires PostgreSQL version 10.
+  - {{ site.data.product.title_short }} requires PostgreSQL version 9.5.
```

Comment:
Ths is wrong and should be 10.

It seems a number of changes in this PR reverted things you meant to keep?

Reply:
I have changed the version to 10.